### ISSUE TITLE
Update Node and Git versions being installed by Windows script

### ISSUE THREAD

**Issue by slkennedy** [unumux/ux-cli#41]

**Comment by slkennedy** [comment]
Updating to v6.6.0 pre-Long Term Support.  New issue created to update Node to LTS version once it is ready in October.

**Comment by chuckhendo** [comment]
Fixed with #43
